client/dashboard/data/index.ts

```diff
@@ -86,6 +86,26 @@ export const fetchPHPVersion = async ( id: string ): Promise< string | undefined
 	} );
 };
 
+export const fetchWordPressVersion = async ( siteIdOrSlug: string ): Promise< string > => {
+	return wpcom.req.get( {
+		path: `/sites/${ siteIdOrSlug }/hosting/wp-version`,
```

Comment:
I couldn't find any docs to this endpoint. Can you share if you have a link that I missed?

Does it just return `latest` or `beta`? Do we need these labels (in design spec or if we deem them useful)?

Can we get this info and the WP version from a single place?

Reply:
There's a whole project for this feature: pesfPa-qt-p2.

Yes, the endpoint just return either `latest` or `beta`. Also, this is how it looks in dashboard v1:

<img width="743" alt="image" src="https://github.com/user-attachments/assets/66cb32eb-b555-4d70-bf5d-a585c48b6dbc" />

I'm really hesitant to change the endpoint (it would break v1 or whatnot) or the core design of this feature for now... porting features to v2 and changing the functionalities should be different activities.

Reply:
>porting features to v2 and changing the functionalities should be different activities.

Agreed, but we should track what we can improve and/or simplify. I might be missing context, but I think we should consider getting WP info from a single endpoint (probably remove this one and update the calls of v1).

Additionally, not for this PR, we might want to revisit the design for this and add some help text about the options.